adafruit_gps.py

```diff
@@ -45,7 +45,7 @@
 import time
 from micropython import const
 
-__version__ = "0.0.0-auto.0"
+__version__ = "3.6.3"
```

Comment:
Was there a reason this was done? I think we should have it set to the current version in github.

Reply:
No reason - accident

Reply:
no worries, just change it back after you run black. Thanks!